src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -312,6 +305,32 @@ function MoneyRequestPreviewContent({
                                                 </Text>
                                             )}
                                         </View>
+                                        {isScanning && (
+                                            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2]}>
+                                                <Icon
+                                                    src={ReceiptScan}
+                                                    height={variables.iconSizeExtraSmall}
+                                                    width={variables.iconSizeExtraSmall}
+                                                    fill={theme.textSupporting}
+                                                />
+                                                <Text style={[styles.textMicroSupporting, styles.colorMuted, styles.ml1, styles.amountSplitPadding]}>
```

Comment:
`colorMuted` here is unnecessary, right? Since we already cover this in the `textMicroSupporting` styles:
![CleanShot 2024-04-15 at 13 27 59@2x](https://github.com/Expensify/App/assets/2319350/8f8e285a-2703-4637-8d58-e9284a04d0c7)